ovos_persona_server/.env.example

```
@@ -0,0 +1,47 @@
+
+# Summarizer plugin settings
+SUMMARIZER_PLUGIN="ovos-summarizer-openai-plugin"
+SUMMARIZER_PROMPT="Your task is to summarize text. Always answer in plaintext with no formatting. Focus on the most important information. ------- {content}"
+# Default to OPENAI_XXX values if not set
+SUMMARIZER_URL=""
+SUMMARIZER_KEY=""
+SUMMARIZER_MODEL=""
+
+# Reranker plugin settings
+RERANKER_PLUGIN="ovos-flashrank-reranker-plugin"
+RERANKER_MODEL="ms-marco-MultiBERT-L-12"
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)